### PR TITLE
ci: add linter on pre-commit hook

### PR DESCRIPTION
**Pull request by totegsito**
### Main changes

- :cloud: (ci): Add linter on pre-commit hook

### CODE REVIEW

**Review by inigomarquinez — COMMENTED** [review]
Thanks for your contribution @totegsito !

I've added a small suggestion just in case you find it useful

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```
[the remark this answers is not in the document]
Comment (totegsito) [reply .husky/pre-commit:4]: No I would not. --fix potentially modifies staged code. I prefer developers to know what changes are being done. Instead, I would add a linting step to github actions

[the remark this answers is not in the document]
Comment (inigomarquinez) [reply .husky/pre-commit:4]: Understood

**Review by inigomarquinez — APPROVED** [review]
LGTM!

**Pull request merged**